### PR TITLE
Remove Duplicate Elements, Fix Naming Inconsistency

### PR DESCRIPTION
**Pull request by colebillys19**
- removed duplicate labels/inputs from settings pane (as seen in last Friday's presentation)

- there was some naming inconsistency regarding the GIF text/caption - in some places it was being referred to as text and others as caption - I changed it to text across the board

- reimplemented the mute preview functionality, it got overwritten at some point

- added some margin space to the bottom of the preview tool, previously there was no space

- made sure all tests were passing

### CODE REVIEW

File: src/components/Preview.js [hunk 1]
```diff
@@ -219,6 +221,11 @@ class Preview extends Component {
             />
           ))}
         </div>
+        {!numFrames ? (
+          <div className="Preview-no-frames">
+            No frames have been captured...
```
Comment (misscoded) [inline src/components/Preview.js]: Why this message instead? Not nearly as helpful in guiding the user to fix the problem.

**Review by misscoded — CHANGES_REQUESTED** [review]
See comment.

File: src/components/Preview.js (hunk 1 above)
Comment (colebillys19) [reply src/components/Preview.js]: Screen Shot 2019-07-03 at 2 17 44 PM
Screen Shot 2019-07-03 at 2 15 24 PM
If you use the frame editor to delete all frames you have the option to undo delete. The bottom screenshot made more sense to me but I may just be overthinking it. (technically you can "add" a frame by hitting undo delete)

Comment (colebillys19) [reply src/components/Preview.js]: but good call, I'll just switch it back

Comment (colebillys19) [reply src/components/Preview.js]: done

**Review by misscoded — APPROVED** [review]
:shipit:

**Pull request merged**